docs/man/git-lfs-migrate.1.ronn

```diff
@@ -80,6 +80,11 @@ options and these additional ones:
 * `--verbose`
     Print the commit oid and filename of migrated files to STDOUT.
 
+* `--object-map=<path>`
+    Write to 'path' a file with the mapping of each rewritten commits. The file
+    format is CSV with this pattern:
+      * OLD-SHA,NEW-SHA
```

Comment:
Can the `*` be removed? Since this is not part of the file format, I think this character has the potential to cause confusion for users trying to understand the file format of the `--object-map` flag.

Reply:
I have reformulated to be on the same line